mkdocs.yml

```diff
@@ -225,6 +225,10 @@ nav:
       - Quick Start: data/sdks/index.md
       - Browser:
         - data/sdks/typescript-browser/index.md
+        - Typescript Browser:
+          - data/sdks/typescript-browser/index.md
+        - Marketing Analytics Browser:
```

Comment:
I know [I voted for option 2](https://amplitude.slack.com/archives/C02E1H87E2C/p1665526332223369), but after seeing the amount of overlap with the standard SDK, I kind of feel option #1 is better.

The main thing I was concerned about is that the Marketing Analytics has it's own side nav item, which we can still do in option #1.